goldstone/core/fixtures/core_initial_data.yaml

```diff
@@ -308,4 +309,39 @@
          }
       }
    }
+}'
+
+  - model: core.AlertSearch
```

Comment:
Remove this from fixtures.  I don't think we want an alert for every openstack error.